src/content/guides/build-performance.md

```diff
@@ -152,6 +155,59 @@ new CommonsChunkPlugin({
 })
 ```
 
+### Avoid Extra Optimization Steps
+
+webpack does extra algorithmic work to optimize the output for size and load performance. These are usually not needed during iterative development:
```

Comment:
What you think about just: `...not needed during development`?

Reply:
I keep "usually" because if it's actually really not needed during development, then why would webpack even have that be the default in development mode? I should probably make it clear somehow that these "optimizations are performant in smaller projects, but not needed in larger codebases"